dai_cadcad/flip.py

```diff
@@ -0,0 +1,134 @@
+""" Flip Module
+    Class-based representation of the Flip smart contract
+    (contains only what is necessary for the simulation)
+
+"""
+from dai_cadcad.pymaker.numeric import Wad, Rad
+from dai_cadcad.util import require
+
+
+class Bid:
+    bid = Rad(0)
+    lot = Wad(0)
+    guy = ""
+    tic = ""  # Date
+    end = ""  # Date
+    usr = ""
+    gal = ""
+    tab = Rad(0)
+    id = ""
+
+    def __init__(self, bid, lot, guy, tic, end, usr, gal, tab, id):
+        self.bid = bid
+        self.lot = lot
+        self.guy = guy
+        self.tic = tic
+        self.end = end
+        self.usr = usr
+        self.gal = gal
+        self.tab = tab
+        self.id = id
+
+
+class Flipper:
+    ADDRESS = ""
+
+    beg = None
+    bids = {}
+    cat = None
+    ilk_id = None
+    kicks = None
+    ttl = None
+    tau = None
+    vat = None
```

Comment:
this is super nitpicky but can we have thee default values be this:
```
beg = Wad(0)
bids = {}
cat = None
ilk_id = ""
kicks = 0
ttl = 0
tau = 0
vat = None
```
reason being that this will indicate the data types of each of these instance variables